tests/test_acceptance.py

```diff
@@ -0,0 +1,466 @@
+            f"Expected project name to be '{project_name}', but got "
+            f"'{actual_project_name}'"
+        )
+
+        # Get the current value of the date input
+        actual_date = target.selenium.execute_script(
+            "return arguments[0].value;", date_input
+        )
+        assert (
+            actual_date == date_value
+        ), f"Expected date to be '{date_value}', but got '{actual_date}'"
+
+        # Get the current value of the time input
+        actual_time = target.selenium.execute_script(
+            "return arguments[0].value;", time_input
+        )
+        assert (
+            actual_time == time_value
+        ), f"Expected time to be '{time_value}', but got '{actual_time}'"
+
+    @pytest.mark.timeout(300)
+    @pytest.mark.asyncio
+    async def test_project_information_iso_format(
+        self,
+        target: Screen,
+        given_app: MultiprocessingWorker,
+        resource_manager: ResourceManager,
+    ) -> None:
+        """Test the project information form with ISO date format.
+
+        This test verifies that the project information form works correctly with
+        ISO date format:
+        1. Setting values for project name, date (2023-05-24), and time
+           (06:00:00) inputs
+        2. Verifying that the values are correctly set in the form
+
+        Note: This test assumes the application is already running from a
+        previous test. When run individually, you may need to start the
+        application first.
+        """
+
+        # Only start the app if it's not already running
+        if not given_app.is_alive():
+            given_app.start()
+        # Open the main page
+        target.open(ENDPOINT_MAIN_PAGE)
+
+        project_name = "Test Project - Leipzig Test Intersection - OTCamera19"
+        date_value = "2023-05-24"
+        time_value = "06:00:00"
+
+        target.find("Project")
+
+        # Find and set the project name input
+        project_name_input = target.find_by_css(
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_PROJECT_NAME)}"]'
+        )
+        set_input_value(target, project_name_input, project_name)
+
+        # Find and set the date input using JavaScript with ISO format
+        date_input = target.find_by_css(
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_DATE)}"]'
+        )
+        set_input_value(target, date_input, date_value)
+
+        # Find and set the time input using JavaScript
+        time_input = target.find_by_css(
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_TIME)}"]'
+        )
+        set_input_value(target, time_input, time_value)
+
+        # Wait a moment for the UI to update
+        target.wait(0.5)
+
+        # Take a screenshot of the page
+        target.shot("project_information_iso_format")
+
+        # Verify that the values were correctly set in the form
+        actual_project_name = target.selenium.execute_script(
```

Comment:
Use `project_name_input.get_attribute("value")` instead. Better would be `target.should_contain_input` which will search until the element is found or a timeout is reached